packages/web/src/types.ts

```diff
@@ -110,6 +110,8 @@ export type rangeLabelsAlign = 'left' | 'right';
 
 export type title = string | any;
 
+export type focusShortcuts = any[];
```

Comment:
@mohdashraf010897 Can you use the property type? I think it should be an array of strings or numbers.